Login/Register/ register.tsx

```diff
@@ -0,0 +1,110 @@
+import React, {useState} from 'react';
+import {
+  SafeAreaView,
+  Text,
+  View,
+  TouchableWithoutFeedback,
+  ScrollView,
+} from 'react-native';
+import {useTailwind} from 'tailwind-rn';
+import {useNavigation} from '@react-navigation/native';
+import {StackNavigationProp} from '@react-navigation/stack';
+import {RootStackparams} from '../../App';
+import AntDesign from 'react-native-vector-icons/AntDesign';
+import Businesfrom from '../action/businessform';
+import Formconfigpassword from '../action/formcomfigpass';
+import Forminput from '../action/fontinput';
+import Formoffline from './Formoffline';
+import Formonline from './Formonline';
+import Checkfrom from '../Register/checkfrom';
+import Checkfrompass from './checkfrompass';
+import Checkfromconfig from './checkconfigpass';
+import Modalhandlecamera from './modalhandlecammera';
+
+const Register = () => {
+  const navigation = useNavigation<StackNavigationProp<RootStackparams>>();
+  const tw = useTailwind();
+  const [display, setdisplay] = useState(false);
+  const [email, setemail] = useState('');
+  const [password, setpassword] = useState('');
+  const [cfp, setcfp] = useState('');
+  const [nshop, setnshop] = useState('');
+  const [openmodal, setopenmodal] = useState(false);
+  const [img, setimg] = useState('');
+
+  return (
+    <SafeAreaView style={tw('flex-1 mx-4 mt-2')}>
+      <ScrollView>
+        {<Formconfigpassword name={'Đăng ký đại lý'} />}
```

Comment:
```suggestion
        <Formconfigpassword name={'Đăng ký đại lý'} />
```
không có điều kiện thì không dùng {}